docs/architecture/governance/amendment-history.md

```diff
@@ -6,15 +6,16 @@ last_update:
 ---
 
 As presented in [Governance](../governance), the Tezos blockchain is constantly evolving, through new amendments.
-These are the past proposals to amend the Tezos protocol.
+
+The history of the Tezos protocol can thus be presented as a list of past amendment proposals that were successfully voted and thus resulted in activated protocols.
```

Comment:
I'm not sure that this sentence add much to tech docs, but if you think it's important, here's a suggestion to simplify it:
```suggestion
These approved amendments form the history of the Tezos protocol:
```